### PR TITLE
promote eck 2.13

### PR DESCRIPTION
**Pull request by kvalliyurnatt**
This adds ECK 2.13 as the current branch. ECK 2.13.0 is planned for 21st May 2024, so this PR should be merged on 21st and after #3002 .

### CODE REVIEW

File: conf.yaml [hunk 1]
```diff
@@ -2145,7 +2145,7 @@ contents:
             prefix:     en/cloud-on-k8s
             tags:       Kubernetes/Reference
             subject:    ECK
-            current:    2.12
+            current:    2.13
             branches:   [ {main: master}, 2.12, 2.11, 2.10, 2.9, 2.8, 2.7, 2.6, 2.5, 2.4, 2.3, 2.2, 2.1, 2.0, 1.9, 1.8, 1.7, 1.6, 1.5, 1.4, 1.3, 1.2, 1.1, 1.0, 1.0-beta, 0.9, 0.8 ]
```
Comment (bmorelli25) [inline conf.yaml:2149]:
```suggestion
            branches:   [ {main: master}, 2.13, 2.12, 2.11, 2.10, 2.9, 2.8, 2.7, 2.6, 2.5, 2.4, 2.3, 2.2, 2.1, 2.0, 1.9, 1.8, 1.7, 1.6, 1.5, 1.4, 1.3, 1.2, 1.1, 1.0, 1.0-beta, 0.9, 0.8 ]
```

File: conf.yaml [hunk 2]
```diff
@@ -2145,8 +2145,8 @@ contents:
             prefix:     en/cloud-on-k8s
             tags:       Kubernetes/Reference
             subject:    ECK
-            current:    2.12
-            branches:   [ {main: master}, 2.12, 2.11, 2.10, 2.9, 2.8, 2.7, 2.6, 2.5, 2.4, 2.3, 2.2, 2.1, 2.0, 1.9, 1.8, 1.7, 1.6, 1.5, 1.4, 1.3, 1.2, 1.1, 1.0, 1.0-beta, 0.9, 0.8 ]
+            current:    2.13
+            branches:   [ {main: master}, 2.13, 2.12, 2.11, 2.10, 2.9, 2.8, 2.7, 2.6, 2.5, 2.4, 2.3, 2.2, 2.1, 2.0, 1.9, 1.8, 1.7, 1.6, 1.5, 1.4, 1.3, 1.2, 1.1, 1.0, 1.0-beta, 0.9, 0.8 ]
```
Comment (bmorelli25) [inline conf.yaml:2149]: Oh I see. You're adding this in a separate PR.
```suggestion
            branches:   [ {main: master}, 2.12, 2.11, 2.10, 2.9, 2.8, 2.7, 2.6, 2.5, 2.4, 2.3, 2.2, 2.1, 2.0, 1.9, 1.8, 1.7, 1.6, 1.5, 1.4, 1.3, 1.2, 1.1, 1.0, 1.0-beta, 0.9, 0.8 ]
```

**Comment by bmorelli25** [comment]
Okay wait I'm confused. If both of these PRs need to be merged on the same day then we should just combine them. I don't see a benefit to merging them separately.

**Comment by kvalliyurnatt** [comment]
> Okay wait I'm confused. If both of these PRs need to be merged on the same day then we should just combine them. I don't see a benefit to merging them separately.
my bad, #3002  will be merged one day before release, on May 20th and this on May 21st

**Review by bmorelli25 — APPROVED** [review]

**Pull request merged**